ext/storage_sources/s3_store/test/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+project(s3_store_tests CXX)    
+include(${CMAKE_SOURCE_DIR}/test/ctest_helpers.cmake)
+
+# Build the unit test binary.
+set(TEST_NAME s3_unit_tests)
+set(TEST_EXE run_s3_unit_tests)
+create_test_executable(${TEST_EXE} 
```

Comment:
np: Since we only anticipate a single test, we can remove the `TEST_NAME` and `TEST_EXE` variables and just substitute the value directly into the `create_test_executable`, `add_test` and `set_tests_properties`